backend/Dockerfile

```diff
@@ -1,23 +1,16 @@
-FROM gradle:6.7.1-jdk11-hotspot
+FROM gradle:6.9.2-jdk11
```

Comment:
Is there an issue moving away from the HotSpot version?

Reply:
It might be nice to have this and Dockerfile.prod stay in sync as far as which underlying image we pull from.

Reply:
AFAICT, hotspot is actually the default JVM, so adding `-hotspot` is redundant.

Reply:
Learn something every day - just checked the list of tags and the hashes are definitely the same.